model/metrics/system-metrics.yaml

```diff
@@ -105,12 +103,24 @@ groups:
   - id: metric.system.memory.usage
     type: metric
     metric_name: system.memory.usage
-    brief: ""
+    brief: "Reports memory in use by state."
+    note: |
+      The sum over all `system.memory.state` values SHOULD equal the total memory
```

Comment:
In principle this is correct but is this a hard a requirement? What if some of the states are not available?

Reply:
It's not a hard requirement, that's why I used 'SHOULD' instead of 'MUST' (so, *there may exist valid reasons in particular circumstances to ignore a particular item*). Do you think that's too strong a requirement?

Reply:
Ok, makes sense then!